deploy/cic-spec.yaml

```
@@ -1043,3 +1043,7 @@ x-amazon-apigateway-request-validators:
     both:
       validateRequestBody: true
       validateRequestParameters: true
+    requestBodyOnly:
+      validateRequestBody: true
+    requestParamsOnly:
+      validateRequestParameters: true
```

nit: let's be explicit here
```suggestion
    requestBodyOnly:
      validateRequestBody: true
      validateRequestParameters: false
    requestParamsOnly:
      validateRequestBody: false
      validateRequestParameters: true
```